lib/mi_bridges/driver/benefits_selector.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class BenefitsSelector < BasePage
+      def fill_in_required_fields
+        click_id("#requestFoodShare")
+        click_id("#isAuthorized_N")
```

Comment:
what does this id represent? worth it to wrap it into a private method for explicitness?